### PR TITLE
adding first version of setup atom ide page

### PR DESCRIPTION
**Pull request by trendzetter**
you might want to edit this version to add some pointers for mac

### CODE REVIEW

File: docs/build/tools/useful-tools.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ import TabItem from '@theme/TabItem';
 ---
 
 - [Chainweaver (Wallet & Workbench)](../../basics/chainweaver/chainweaver-user-guide/)
-- [Atom IDE](https://atom.io)
+- [Atom IDE](atom-ide)
```
Comment (JCM00N) [inline docs/build/tools/useful-tools.md:14]: Is this line still necessary, when the link to Atom is in the toolbar itself and is distinct from the "Other" section?

**Comment by Randynamic** [comment]
@trendzetter TY for your contribution. I do have one request can we get this PR more aligned with [this article](https://github.com/kadena-io/pact#atom) so we can merge this PR 😉 .

**Comment by trendzetter** [comment]
@Randynamic  I think the linked page is a bit confusing. Could you specify which elements are desirable to include in the tutorial. I don't think it makes sense for a practical tutorial for beginners to include all editors and compile from sources instructions.

**Comment by Randynamic** [comment]
@trendzetter, my bad I had to be more specific. I'm talking about the installment of Pact itself. IMHO It's better to just point out to that part of the documentation how to install Pact, as there are more ways to do this. We don't want to have conflicting or multiple explanations on how the installment should be (also not from a maintenance point of view).

**Comment by trendzetter** [comment]
@Randynamic I took your comment as an opportunity to rework my pr so it fits in the existing pact tutorial. In the meantime I could cut out the redundancy. So there is no separate page anymore in my pr, I just add on the page: learn-pact/beginner/atom-sdk in order to have a broader application of the tutorial. Instead of only mac if talks takes into account other desktop OS scenarios.
On the page Atom SDK there is a reference to the pact README you specified on your comment.
thanks for the feedback!

**Comment by Randynamic** [comment]
@trendzetter thank you for your contribution 👍

**Pull request merged**